src/main/java/peoplesoft/model/employment/Employment.java

```diff
@@ -23,51 +26,84 @@
 
 import peoplesoft.commons.util.JsonUtil;
 import peoplesoft.model.Model;
+import peoplesoft.model.employment.exceptions.DuplicateEmploymentException;
+import peoplesoft.model.employment.exceptions.EmploymentNotFoundException;
 import peoplesoft.model.job.Job;
-import peoplesoft.model.person.Name;
 import peoplesoft.model.person.Person;
+import peoplesoft.model.util.ID;
 
 /**
  * Association class to handle assigning {@code Jobs} to {@code Persons}.
  */
 @JsonSerialize(using = Employment.EmploymentSerializer.class)
 @JsonDeserialize(using = Employment.EmploymentDeserializer.class)
 public class Employment {
-    // TODO: Refactor class name/package if necessary
-    // TODO: Feel free to change implementation
     /**
      * Singleton instance.
      */
     private static Employment instance;
 
     /**
-     * Maps {@code JobId} to {@code Name}.
+     * Maps {@code JobId} to {@code PersonId}.
      */
-    private HashMap<ID, Name> map;
+    private HashMap<ID, List<ID>> map;
```

Comment:
- `Set<ID>` might be more appropriate than `List<ID>`
- should we use the `Map` interface instead of `HashMap`?

Reply:
Perhaps, but `List` preserves insertion order. Do you think that is needed?

Reply:
doesn't seem too necessary, but if we need to preserve insertion order then maybe a `LinkedHashSet`? just to communicate that we only ever expect one occurrence of any given personID for each jobID